### PR TITLE
Remove stdlibs from blacklist

### PR DESCRIPTION
**Pull request by gdbelvin**
Remove stdlibs from blacklist

Reverts f92eb83a0581d00739856551ff59a1ccecb3e2c4
Follows #2032  #2030

### CODE REVIEW

File: .golangci.yaml [hunk 1]
```diff
@@ -14,6 +14,7 @@ linters-settings:
       - github.com/gogo/protobuf/proto
       - github.com/google/certificate-transparency-go/trillian
       - github.com/google/certificate-transparency-go/x509
+      - github.com/google/certificate-transparency-go/asn1
       - encoding/asn1
       - crypto/x509
```
Comment (RJPercival) [inline .golangci.yaml]: Should these stdlib packages be on the block list?

Comment (RJPercival) [reply .golangci.yaml]: I wonder why they weren't actually being blocked? You've recently added crypto/x509 imports without complaint from the linter right?

Comment (gdbelvin) [reply .golangci.yaml]: The linter only runs against *new* code.

Comment (RJPercival) [reply .golangci.yaml]: Those imports you added were new though, weren't they?

**Pull request merged**